src/main/java/Duke.java

```diff
@@ -100,7 +100,7 @@
     public static final int NUM_LOGOS_AVALIABLE = 4;
 
     // array containing all tasks the user has input
-    private static ArrayList<Task> allTasks = new ArrayList<>(MAX_TASKS);
+    private static ArrayList<Task> allTasks = new ArrayList<Task>(MAX_TASKS);
 
     private static int numTasks = 0;
 
```

Comment:
In general very good following of coding standard for the course: all caps for constands, camelCase for variables and methods etc..